cmd/test_load.go

```diff
@@ -265,11 +265,19 @@ func loadSystemCertPool(logger logrus.FieldLogger) {
 func (lct *loadedAndConfiguredTest) buildTestRunState(
 	configToReinject lib.Options,
 ) (*lib.TestRunState, error) {
-	// This might be the full derived or just the consodlidated options
+	// This might be the full derived or just the consolidated options
 	if err := lct.initRunner.SetOptions(configToReinject); err != nil {
 		return nil, err
 	}
 
+	// Here, where we get the consolidated options, is where we check if any
+	// of the deprecated options is being used, and we report it.
+	if _, isPresent := configToReinject.External["loadimpact"]; isPresent {
+		if err := lct.preInitState.Usage.Strings("deprecated_options", "ext.loadimpact"); err != nil {
```

Comment:
In https://github.com/grafana/k6/pull/4030/files the key is `deprecations/<what is deprecated>`. I would prefer if we keep the keys more generic and `deprecated_options` is very specific to options by the looks of it and I do not know of any other option that will make it relevant to have a separate key.

Reply:
So, is your suggestion: `deprecations/ext.loadimpact`? 
I'm happy with that one as well, sure!

Reply:
I guess that looks okay - yes. Don't know if we want `options` somewhere in there, but as a one of case this seems good enough.

Reply:
I've applied the feedback to get it mergable